client/src/main/scala/drt/client/components/PortExportDashboardPage.scala

```diff
@@ -20,7 +20,7 @@ object PortExportDashboardPage {
           <.div(config.terminals.map(tn => {
             <.div(
               <.h3(s"Terminal $tn"),
-              MultiDayExportComponent(tn, SDate.now(), p.loggedInUser)
+              MultiDayExportComponent(tn, SDate.now(), p.loggedInUser,false)
```

Comment:
we have been trying to add named parameters for boolean values - I think IntelliJ suggests it if you alt+enter on here.